x-pack/legacy/plugins/actions/server/actions_config.ts

```diff
@@ -65,14 +82,26 @@ function isWhitelistedHostnameInUri(config: ActionsConfigType, uri: string): boo
   );
 }
 
+function isActionTypeEnabledinConfig(
+  { enabledTypes }: ActionsConfigType,
+  actionType: string
+): boolean {
+  const enabled = new Set(enabledTypes);
+  if (enabled.has('*')) return true;
```

Comment:
Can we use the enum here?

Reply:
woops, ya, I added the enum after starting with just using literal `*` - should be using the enum

Reply:
fixed